docs/modules/ROOT/pages/explanations/decisions/mongodb.adoc

```diff
@@ -0,0 +1,37 @@
+= MongoDB
+
+== Problem
+
+We want to Provide MongoDB via AppCat.
+But their licensing might be an issue.
+This decision looks closely at the licensing and alternative products that can be used as drop-in replacements for MongoDB.
+
+== Proposals
+
+Proposal 1: Use MongoDB Community Edition::
+MongoDB community edition is licensed under the https://en.wikipedia.org/wiki/Server_Side_Public_License[SSPL].
```

Comment:
citation needed (on how CE is licensed)

Reply:
Probably https://github.com/mongodb/mongo/blob/master/LICENSE-Community.txt

Reply:
Added the link to the docs.